daemon/src/maker_cfd.rs

```diff
@@ -75,8 +75,57 @@ impl Actor {
     ) -> Result<Self> {
         let mut conn = db.acquire().await?;
 
+        let cfds = load_all_cfds(&mut conn).await?;
+
         // populate the CFD feed with existing CFDs
-        cfd_feed_actor_inbox.send(load_all_cfds(&mut conn).await?)?;
+        cfd_feed_actor_inbox.send(cfds.clone())?;
```

Comment:
Maybe it's better to move those state changes out of this actor after all - has to many side effects in here. I will see to move it to a `cleanup` module. I had that initially but decided it would be better to move this into the actor, because the cfd actor should be responsible for all state transitions...